src/core/lib/support/murmur_hash.h

```diff
@@ -38,6 +38,21 @@
 
 #include <stddef.h>
 
+#ifdef __hpux
```

Comment:
As in the other cases, this should use a constant defined in `port_platform.h`. In this particular case, I would recommend something like `GPR_LITTLE_ENDIAN`.